### ISSUE TITLE
Allow reading specific Cop documentation from command-line

### ISSUE THREAD

**Issue by brodock** [rubocop/rubocop#7671]
## Is your feature request related to a problem? Please describe.

There are a few situations where you have only the Cop name but you are not able to figure out what it is enforcing. Here are a few examples:
* CI fails 
* Someone disabled a cop inline on the source code
* A new issue is discussing enabling/disabling a specific Cop

## Describe the solution you'd like

It would be useful to have a command on the command line that allowed me to paste the cop and get a simple documentation. (so it can be offline friendly).

Here is a minimal implementation:

```
$ rubocop show Lint/SafeNavigationConsistency

This cop check to make sure that if safe navigation is used for a method call in an && or || condition that safe navigation is used for all method calls on that same object.
```

An improved implementation:

```
$ rubocop show Lint/SafeNavigationConsistency

This cop check to make sure that if safe navigation is used for a method call in an && or || condition that safe navigation is used for all method calls on that same object.

 - Enabled? Yes/No (default: Yes)
 - Safe? Yes
 - Supports autocorrection? Yes

Configurable attributes:
  - AllowedMethods: Array (default: [present?, blank?, presence, try, try!])
```

## Describe alternatives you've considered

An alternative would be to link to the external documentation pointing to the right page, for example: https://www.rubocop.org/en/latest/cops_lint/#lintsafenavigationconsistency (not offline friendly)

**Comment by bbatsov** [comment]
Good idea!

**Comment by scottmatthewman** [comment]
We already have `--show-cops` on the command line, which does at least show the current configuration. For example:

```bash
$ rubocop --show-cops Lint/SafeNavigationConsistency
# Supports --auto-correct
Lint/SafeNavigationConsistency:
  Description: Check to make sure that if safe navigation is used for a method call
    in an `&&` or `||` condition that safe navigation is used for all method calls on
    that same object.
  Enabled: true
  VersionAdded: '0.55'
  VersionChanged: '0.77'
  AllowedMethods:
  - present?
  - blank?
  - presence
  - try
  - try!
```

This might be a good place to start, and see what other information would be good to include?

**Comment by dvandersluis** [comment]
A potential problem here is that we'd have to generate the yarddoc for the cop on the fly, which is a bit slow. I'm thinking about how to implement this.
